### ISSUE TITLE
show the clan's rating details when hovering over a move

### ISSUE THREAD

**Issue by jankei** [Backfeed/ChessMates#18]
blocked by #19

**Comment by ghost** [comment]
In the current UI design draft, clan's rating details are:
- weighed rating
- confidence index
- number of evaluations
- Weighed rating: (sum(userRating \* userReputation))/(sum(userReputation) - sorry about the format, I know it's not a proper formula, but you get it right? :) Help anyone to express it properly?
- Confidence index: (sum the reputation of users that have rated the move) / (reputation of all the users that have played the previous move, + any player that start to evaluate or suggest move during the turn)
  In the case of the first move, I suggest that the total reputation is based on the number of authentified clan members currently connected.

NB: More generally, the question of defining the total reputation at any moment during a game is complicated. If we take into account connected clan members, we might include people that are sleeping players (with an open window on a game, but without playing). If we only take into account people that have already played (by suggesting or evaluating a move), then the confidence index may be completely erratic, starting at a very high value for the first moves, even when a very low clan reputation is engaged. That's why I suggest to start with an approximative value (players of the previous move, or players connected for the first move) that is more likely to close to the reality.

Does it make sense @yaoe ?

**Comment by goldylucks** [comment]
duplicate of #31
